e2e-tests/functions

```diff
@@ -16,7 +16,7 @@ SKIP_BACKUPS_TO_AWS_GCP_AZURE=${SKIP_BACKUPS_TO_AWS_GCP_AZURE:-1}
 PMM_SERVER_VER=${PMM_SERVER_VER:-"9.9.9"}
 IMAGE_PMM_CLIENT=${IMAGE_PMM_CLIENT:-"perconalab/pmm-client:dev-latest"}
 IMAGE_PMM_SERVER=${IMAGE_PMM_SERVER:-"perconalab/pmm-server:dev-latest"}
-CERT_MANAGER_VER="1.15.3"
+CERT_MANAGER_VER="1.16.1"
```

Comment:
Hi, we need to bump cert manager versions also in go files like: https://github.com/percona/percona-server-mongodb-operator/pull/1643/files

Reply:
done

Reply:
So, I reverted cert-manager back to v1.15.3 as we discussed. The version bump will be in a separate PR.